### ISSUE TITLE
text embedding dim(2048) not match with codec dim(1024) with Qwen3-TTS-12Hz-0.6B-Base

### ISSUE THREAD

**Issue by HCRTY** [QwenLM/Qwen3-TTS#120]
### Description

I try finetuning Qwen3-TTS-12Hz-0.6B-Base on single speaker, but I get error: text embedding size do not match with audio codec embedding size

<img width="1252" height="69" alt="Image" src="https://github.com/user-attachments/assets/eccb664c-ff03-4b54-bd7a-ce1f72b75492" />

### Reproduction

accelerate launch --num_processes 8 sft_12hz.py \
  --init_model_path Qwen3-TTS-12Hz-0.6B-Base \
  --output_model_path exp \
  --train_jsonl data.jsonl \
  --batch_size 8 \
  --lr 2e-5 \
  --num_epochs 10 \
  --speaker_name test

### Logs

```shell

```

### Environment Information

NVIDIA GPU

### Known Issue

- [x] The issue hasn't been already addressed in Documentation, Issues, and Discussions.

**Comment by xiaoyangnihao** [comment]
@fumyou13 can you fintune on multi-gpus ?
